### PR TITLE
adding isvalidnext and tests

### PR DESCRIPTION
**Pull request by jiyoonie9**

### CODE REVIEW

File: tbdex/rfq/rfq_test.go [hunk 1]
```diff
@@ -425,3 +430,25 @@ func TestVerify_SignedWithWrongDID(t *testing.T) {
...
+		offeringID.String(),
+		rfq.Payin("100", "STORED_BALANCE"),
+		rfq.Payout("BANK_ACCOUNT"),
+	)
+
+	assert.False(t, r.IsValidNext(rfq.Kind))
+	assert.False(t, r.IsValidNext(order.Kind))
+	assert.False(t, r.IsValidNext(orderstatus.Kind))
+
+	// todo rfq can only be followed by quote... or close?
```
Comment (jiyoonie9) [inline tbdex/rfq/rfq_test.go]: can rfq be followed by a quote or a close? or only a quote?

Comment (KendallWeihe) [reply tbdex/rfq/rfq_test.go]: rule of thumb is, close is valid after any message kind

**Review by KendallWeihe — APPROVED** [review]

**Review by mistermoe — APPROVED** [review]
naisuuuuu!

File: tbdex/rfq/rfq_test.go (hunk 1 above)
Comment (jiyoonie9) [reply tbdex/rfq/rfq_test.go]: except for close itself! 🤓

**Pull request merged**